Sources/AnyImageKit/Capture/CaptureOptionsInfo.swift

```diff
@@ -11,108 +11,64 @@ import AVFoundation
 
 public struct CaptureOptionsInfo {
     
-    /// 主题色
-    /// 默认：绿色 0x57BE6A
+    /// Tint color.
+    /// - Default: Green
     public var tintColor: UIColor = UIColor.color(hex: 0x57BE6A)
     
-    /// 媒体类型
-    /// 默认：Photo+Video
+    /// Media options for capture.
+    ///
+    /// You can tap to take photo and hold to take video.
+    ///
+    /// - Default: [photo, video]
     public var mediaOptions: CaptureMediaOption = [.photo, .video]
     
-    /// 照片拍摄比例
-    /// iPadOS 环境下无效
-    /// 默认：4:3
+    /// The ratio of take photo.
+    ///
+    /// - Note: Invalid on iPad.
+    ///
+    /// - Default: 4:3
     public var photoAspectRatio: CaptureAspectRatio = .ratio4x3
     
-    /// 使用的摄像头
-    /// iPadOS 环境下仅用于先打开前置/后置摄像头，用户仍然可以切换摄像头
-    /// 默认：后置+前置
+    /// Preferred position of capture.
+    ///
+    /// The camera of the first element of position will be used when open capture
+    /// Eg. [back] Capture will always use back camera, user CAN NOT switch camera.
+    /// Eg. [back, front] The back camera will be used when open capture, then user can switch to front camera.
+    ///
+    /// - Note: On iPad, Capture CAN NOT limit user to switch the camera even though you set one position only. For example set [back] in this case. Capture will use back camera at the first time, then user still can switch camera.
+    ///
+    /// - Default: [back, front]
     public var preferredPositions: [CapturePosition] = [.back, .front]
     
-    /// 默认闪光灯模式
-    /// 默认：关闭
+    /// Flash Mode.
+    /// - Default: off
     public var flashMode: CaptureFlashMode = .off
     
-    /// 视频拍摄最大时间
-    /// 默认 20 秒
+    /// The maximum time(second) of the video can be recorded.
+    /// - Default: 20 second
     public var videoMaximumDuration: TimeInterval = 20
     
-    /// 相机预设
-    /// iPadOS 环境下无效
-    /// 默认支持从 1920*1080@60 开始查找支持的最佳分辨率
+    /// The preferred presets of the camera.
+    ///
+    /// Capture will use the best preset in `preferredPresets` as the camera preset.
```

Comment:
Capture will check camera's support format and chose the best one form `preferredPresets`